src/Symfony/Component/Ldap/LdapClient.php

```diff
@@ -54,6 +54,8 @@ public function __construct($host = null, $port = 389, $version = 3, $useSsl = f
         $this->useSsl = (bool) $useSsl;
         $this->useStartTls = (bool) $useStartTls;
         $this->optReferrals = (bool) $optReferrals;
+
+        $this->connect();
```

Comment:
I don't know, the returned resource is required by 99% of ldap_* function.

If you want, i can put as was.